web/pandas_web.py

```diff
@@ -223,7 +224,23 @@ def home_add_releases(context):
         with open(pathlib.Path(context["target_path"]) / "releases.json", "w") as f:
             json.dump(releases, f, default=datetime.datetime.isoformat)
 
-        for release in releases:
+        # Map to create a list of (version, realse) pairs from releases
+        parsed_releases = map(lambda r: (version.parse(r["tag_name"]), r), releases)
+
+        # Sort the releases in descending order
```

Comment:
I don't think this comment adds much value, as what it's saying can be seen in the code.

Also, just a personal preference, but I'd do the sorting directly when you're looping, `for version in sorted(releases, reverse=True):`. I find it more readable, but also fine if you have a preference for having it separate.